.github/workflows/main.yml

```
@@ -0,0 +1,38 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: CI
```

```suggestion
name: Build and Test
```

Do we want to standardize this jobs name? I feel like I see it most often as "Build and Test"